src/amo/css/AddonDetail.scss

```diff
@@ -1,16 +1,12 @@
+@import "~core/css/inc/mixins";
 @import "~core/css/inc/vars";
 
 .AddonDetail {
-  header {
-    background: $masthead-color;
-    color: $header-font-color;
-  }
-
   hr {
     margin: 0 20px;
   }
 
-  .icon,
+  .AddonDetail-icon,
```

Comment:
I think the other rules still need to be nested so I'll leave this here until those are brought up to date with our more specific class names.